migrations_data/new_project_phase_list_items.js

```diff
@@ -0,0 +1,74 @@
+module.exports = [{
+    "type" : "projectPhase",
+    "_schemaName" : "List",
+    "name" : "Pre-EA"
+  },{
+    "type" : "projectPhase",
+    "_schemaName" : "List",
+    "name" : "Pre-Application"
+  },{
+    "type" : "projectPhase",
+    "_schemaName" : "List",
+    "name" : "Evaluation"
+  },{
+    "type" : "projectPhase",
+    "_schemaName" : "List",
+    "name" : "Application Review"
+  },{
+    "type" : "projectPhase",
+    "_schemaName" : "List",
+    "name" : "Further Assessment"
+  },{
+    "type" : "projectPhase",
+    "_schemaName" : "List",
+    "name" : "Referral"
+  },{
+      "type" : "projectPhase",
+      "_schemaName" : "List",
+      "name" : "Termination"
+  },{
+      "type" : "projectPhase",
+      "_schemaName" : "List",
+      "name" : "Withdrawal"
+  },{
+      "type" : "projectPhase",
+      "_schemaName" : "List",
+      "name" : "Post Decision - Pre-Construction"
+  },{
+      "type" : "projectPhase",
+      "_schemaName" : "List",
+      "name" : "Post Decision - Construction"
+  },{
+      "type" : "projectPhase",
```

Comment:
Can we fix the alignment of items here?

Reply:
Yes thanks, done. Also renamed to be camel case style of file name.